### PR TITLE
More rearranging

### PR DESCRIPTION
**Pull request by overwritten**
(taken from Discord)
Some regex fixes
More in-game translations (Jizou statue relationship panel, side buttons, etc)
Tweaked interaction menu translations a teeny bit
Final maker translation cleanup (for now)
Split Menu into Menu_01 (pre-in game menus) and Menu_02 (in game menus)
All popup menus should now have translations
Changed 'Brave' personality to be more accurate to the kanji -- her new name is 'Hesitant'
Changed 'Shrine Path' to 'River' since that's where you go
Translated all H positions! Woohoo!
Included some dialogue strings from user misc that didn't make it into the hardmod update

### CODE REVIEW

File: Translation/en/Text/regex.txt [hunk 1]
```diff
@@ -35,6 +35,8 @@ r:"^付け毛([０-９0-9]+)$"=Hair $1
 r:"^付け毛([０-９0-9]+)（左）$"=Hair $1 (Left)
 r:"^付け毛([０-９0-9]+)（右）$"=Hair $1 (Right)
 sr:"^エンブレム(\w)$"=Emblem $1
-
+sr:"^(.+)との関係$"=Relationship with $1
+sr:"^([０-９0-9]+)日$"=$1 days
```
Comment (GeBo1) [inline Translation/en/Text/regex.txt]:
```suggestion
r:"^([1１])日$"=$1 day
r:"^([０-９0-9]+)日$"=$1 days
```

File: Translation/en/Text/regex.txt [hunk 2]
```diff
@@ -10,9 +10,9 @@ sr:"^(.+)目 首の向き$"=$1 Eye orientation
 sr:"^会いに(.+)$"=$1 to meet
 sr:"^(.+)の運勢$"=$1 fortune
 sr:"^すごく(.+)！$"=Very $1!
-sr:"^告白して(.+)$"=Confess, $1.
-sr:"^告白(.+.)側$"=Confessed side
-sr:"^一緒に行動(.+)回数$"=Sex together
+sr:"^告白して(.+)$"=No confession yet
+sr:"^告白(.+.)側$"=Confession
+sr:"^一緒に行動(.+)回数$"=Interacted With
```
Comment (GeBo1) [inline Translation/en/Text/regex.txt]: What's the captured (`(.+)`) bit here?  If you're discarding it intentionally (no `$1`) change from `sr:` to `r:`.  Also what's up with `(.+.)`?

File: Translation/en/Text/Dialogue/common.txt [hunk 3]
```diff
@@ -0,0 +1,379 @@
+何人かがいなくなったようだ……=It seems like some people have disappeared...
+間一髪のところで助かった……=We were saved in the nick of time...
+
+ガツガツ= Serious
+大まかに= Quick
+ホドホド= Moderate
+普通に= Normal
+ソコソコ= Quick
+効率的に= Efficient
```
Comment (GeBo1) [inline Translation/en/Text/Dialogue/common.txt]: Should these leading spaces be here?

File: Translation/en/Text/regex.txt (hunk 2 above)
Comment (ManlyMarco) [reply Translation/en/Text/regex.txt]: Yeah these 3 look not right to me, why discard the capture?

Comment (ManlyMarco) [reply Translation/en/Text/regex.txt]:
```suggestion
```
These shouldn't be regexes as it turns out, will be re-added later.

**Pull request merged**